docs/content/stable/architecture/transactions/read-committed.md

```diff
@@ -1557,7 +1555,11 @@ Read Committed interacts with the following feature:
 
 ## Limitations
 
-Refer to [#13557](https://github.com/yugabyte/yugabyte-db/issues/13557) for limitations.
+* A `SET TRANSACTION ISOLATION LEVEL ...` statement immediately issued after `BEGIN;` or `BEGIN TRANSACTION ISOLATION LEVEL READ COMMITTED;` will fail if YB-Tserver gflag `yb_enable_read_committed_isolation=true`. The following error will be thrown: `ERROR:  SET TRANSACTION ISOLATION LEVEL must not be called in a subtransaction` (refer [#12494](https://github.com/yugabyte/yugabyte-db/issues/12494)).
+
+* Read restart and serialization are not internally handled in Read Committed isolation if the query's response size exceeds the YB-TServer gflag `ysql_output_buffer_size` which has a default value of 256KB (refer [#11572](https://github.com/yugabyte/yugabyte-db/issues/11572)).
+
+* Non-transactional side-effects can occur more than once when a conflict/ read restart occurs in functions/ procedures in read committed isolation. This is because in read committed isolation, the retry logic in the database will undo all work done as part of that statement until the error and re-attempt the whole client-issued statement. (refer [#12958](https://github.com/yugabyte/yugabyte-db/issues/12958))
```

Comment:
The statement - "This is because in read committed isolation, the retry logic in the database will undo all work done as part of that statement until the error...." it seems incomplete. What about the error ? 
Can you update that.

Reply:
fixed